api/log/create_service.go

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+//nolint:dupl // ignore similar code to step
+package log
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/go-vela/server/database"
+	"github.com/go-vela/server/router/middleware/build"
+	"github.com/go-vela/server/router/middleware/org"
+	"github.com/go-vela/server/router/middleware/repo"
+	"github.com/go-vela/server/router/middleware/service"
+	"github.com/go-vela/server/router/middleware/user"
+	"github.com/go-vela/server/util"
+
+	"github.com/go-vela/types/library"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
```

Comment:
Could we remove the blank lines between github imports and sort them? Same for the other files.